index.ts

```diff
@@ -28,24 +28,68 @@ function printSbtTask(task: string, cwd?: string): Promise<string> {
       if (code !== 0)
         reject(new Error(`sbt invocation for Scala.js compilation failed with exit code ${code}.`));
       else
-        resolve(fullOutput.trimEnd().split('\n').at(-1)!);
+        resolve(fullOutput.trimEnd().split('\n').slice(-tasks.length));
```

Comment:
Maybe we can. I am not sure what to do there, though. We could filter lines starting with `[info]` (and warning etc.) and assume that this is not the actual path. (Technically, many weird characters might occur there, IIRC Linux allows everything but `'\0'` and `'/'` in a file name. I am sure newlines would probably break it, too, so it depends how much we want to be perfectionists…)

Other alternative would be calling SBT twice, but this would increate startup time, as starting two SBT instances in parallel sometimes fails. This still wouldn't resolve the issue with newlines, so maybe filtering `[info]` and similar prefixes would be a better way.